lib/data/external/AwsClient.js

```diff
@@ -402,6 +402,52 @@ class AwsClient {
             return callback(null, destAwsKey);
         });
     }
+    uploadPartCopy(request, awsSourceKey, sourceLocationConstraintName,
+      log, callback) {
+        const destBucketName = request.bucketName;
+        const destObjectKey = request.objectKey;
+        const destAwsKey = this._createAwsKey(destBucketName, destObjectKey,
+        this._bucketMatch);
+
+        const sourceAwsBucketName =
+        config.locationConstraints[sourceLocationConstraintName]
```

Comment:
we should have a method in Config.js that handles this.  we shouldn't be digging this deep into the object outside of the config.

Reply:
Totally agree!